doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -50,6 +50,14 @@ To configure the SED wake-up intervals, set both of the following Kconfig option
 * :kconfig:option:`CONFIG_CHIP_SED_IDLE_INTERVAL` to a value in milliseconds that determines how often the device wakes up to poll for data in the idle state (for example, ``1000``).
 * :kconfig:option:`CONFIG_CHIP_SED_ACTIVE_INTERVAL` to a value in milliseconds that determines how often the device wakes up to poll for data in the active state (for example, ``200``).
 
+You can also configure the SED active threshold that makes the device to stay active longer after the network activity, instead of going to idle state immediately.
```

Comment:
```suggestion
You can also configure the SED active threshold to make the device stay active after the network activity, instead of going to idle state immediately.
```